make/go/dep_jq.mk

```diff
@@ -9,28 +9,30 @@ $(call _assert_var,CACHE_VERSIONS)
 $(call _assert_var,CACHE_BIN)
 
 # Settable
-# https://stedolan.github.io/jq/download checked 20230201
-JQ_VERSION ?= 1.6
+# https://jqlang.github.io/jq/download/ 20230906 checked 20230907
+JQ_VERSION ?= 1.7
 
-# jq does not have an ARM release on Github so we'll use
-# the amd64 version via Rosetta
 ifeq ($(UNAME_OS),Darwin)
-JQ_OS := osx
-JQ_ARCH := -amd64
+JQ_OS := macos
+ifeq ($(UNAME_ARCH),arm64)
+JQ_ARCH := arm64
+else
```

Comment:
Look at protoc, we havent done `else` in this case if I remember correctly

Reply:
Pushed an update - is that what you meant?